gatsby/util/build-netlify-preview-url.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Get the Netlify Preview URL by previewId
+ * (this equals the github entity id, e.g. PR id).
+ * If no previewId is given the fallback URL gets return.
+ *
+ * On netlify the deployment urls follow a strict pattern:
+ * - https://deploy-preview-${PREVIEW_ID}--${DOMAIN_NAME}.netlify.app/
```

Comment:
This seems to be slightly out of sync with the code ("preview" vs. "review", "fallback URL"). I claim that the tests nicely describe how this function should behave and suggest completely removing this comment.